docs/how-to/create-initial-data-migrations.md

```diff
@@ -0,0 +1,78 @@
+# Create initial data scripts
+
+## Overview
+
+The goal is to convert our initial data into scripts that can be loaded into the database when the backend is set up for the first time.
+
+These are the steps:
+
+   1. Export the data into JSON
+   1. Generate a python script from the JSON data
+
+### Prerequisites
+
+The initial data exists in a Google spreadsheet, such as [this one for People Depot][pd-data-spreadsheet]. There should be individual sheets named after the model names the data correspond to, such as `SOC Major - Data`. The sheet name is useful for us to identify the model it corresponds to.
+
+The sheet should be formatted like so:
+
+- the first row contains the names of the field names in the model. The names must be exactly the same
+- rows 2 to n are the initial data for the model we want to turn into a script.
+
+## Convert the data into JSON
+
+   1. Export the data from the Google [spreadsheet][pd-data-spreadsheet]
+      1. Find the sheet in the document containing the data to export. Let's use the `SOC Major - Data` data as our example. It's
+      1. Make sure that the first row (column names) is frozen. Otherwise, freeze it by selecting the first row in the sheet, then Menu > View > Freeze > Up to row 1
+      1. Export to JSON. Menu > Export JSON > Export JSON for this sheet
```

Comment:
Thanks for the documentation and great comments/questions. But the answers are not going to be great.

1. How much of the above process should be documented?
   - Documentation is great, but this is normal general google apps scripts behavior so it's probably not up to us (as a project) to host this documentation. Maybe someone has the documentation that we can link to or we can put it in a hfla general developer docs place.
   - The other thing is that this is a temporary solution that we'll get rid of soon after this is merged. We won't need/want this script at all. But this is a working solution and I want to get something working before doing the optimizing step.

1. What would be a better way to handle this Google hasn't verified this app pop-up?
   - There is no better way to do this. This is normal and the warning serves to tell users to examine the script closely before running it.

1. What is the benefit of this custom script?
   - Good question. The benefit it that it works to get us JSON from the spreadsheet. Actually, Bonnie found it so it's "free" for me. The drawback is that it's a manual step and copying the JSON into a file is also manual. That's why it only serves as a stop gap solution, and because there's a much better method, I didn't spend time to look for other similar things. The one you found does go one step further than the current script, so I would have gone with that if I had known.

Reply:
Is there an issue for this?  I can work on that if you want.